cmd/zed/import.go

```diff
@@ -164,13 +169,24 @@ func importRelationships(client *authzed.Client, relationships string, definitio
 		return err
 	}
 
-	request := &v1.WriteRelationshipsRequest{Updates: relationshipUpdates}
-	log.Trace().Interface("request", request).Msg("writing relationships")
-	log.Info().Int("count", len(relationshipUpdates)).Msg("importing relationships")
+	log.Info().
+		Int("batch_size", batchSize).
+		Int("workers", workers).
+		Int("count", len(relationshipUpdates)).
+		Msg("importing relationships")
 
-	if _, err := client.WriteRelationships(context.Background(), request); err != nil {
-		return err
-	}
+	err := grpcutil.ConcurrentBatch(context.Background(), len(relationshipUpdates), batchSize, workers, func(ctx context.Context, no int, start int, end int) error {
```

Comment:
In theory we could use a cancel context here and have another cli flag to dictate how long a timeout should be. I think that should be a follow up feature though.